README.md

```diff
@@ -1,38 +1,130 @@
+<div align="center">
+
 # Product Roadmap
 
 ![roadmap-1152x672-flip](https://user-images.githubusercontent.com/194400/27068309-5b379538-5007-11e7-99fb-20f09369c672.jpg)
 
+</div>
+<br />
 
-## _Why_?
+# _Why_? 🤷‍♀️
```

Comment:
because the hierarchy of the headings (from "##" to "#") has changed, we also need to update the other sections (Who, When, How)

Reply:
@SimonLab agreed. If you have time, please update. 📝

Reply:
It should be fixed with https://github.com/dwyl/product-roadmap/pull/22/commits/0862462ef4d7c39f04bead23ecb6a55d08a19c1a

Reply:
Cool. I will ignore the notifications till you're done. 👍

Reply:
:+1: I think I'm almost done with formatting. I've been reading the document and I'll add comments/issue focused on the content itself.

Reply:
Defo ask questions and even open issues for things that are unclear. 👍

Reply:
Great exploratory question: https://github.com/dwyl/product-roadmap/issues/27 Thanks @SimonLab 🎉